src/lib.rs

```diff
@@ -12,6 +12,7 @@ These macros are not intended to be used on fields which require custom logic in
 extern crate getset;
 
 #[derive(Getters, Setters, MutGetters, Default)]
+#[get = "pub"] #[set = "pub"] #[get_mut = "pub"]
```

Comment:
Can you instead add another example that explicitly describes how the struct-level option interacts with field level options. I think this can highlight the feature and caveats better.

Reply:
I think compile_fail example perfect for that